grails-app/services/streama/SettingsService.groovy

```diff
@@ -128,4 +132,20 @@ class SettingsService {
       resultValue.message = "Invalid API language: The entered language is not an IETF language tag.";
     }
   }
+
+  def validateCredentialsForOpensubtitles(Settings settingsInstance, resultValue) {
+    def credentials = settingsInstance.value
+    def url = opensubtitlesService.buildUrl(null, "forrest gump", null, "eng")
+    def response = opensubtitlesService.sendRequest(url, credentials)
+    if (response.statusCodeValue == 403) {
+      resultValue.success = true
+      resultValue.message = "Invalid credentials for opensubtitles: Username or password is incorrect."
+    } else if (response.statusCodeValue == 200) {
+      resultValue.sucess = true
+      resultValue.message = "Credentials for opensubtitles is valid and can be used!"
+    } else {
+      resultValue.sucess = true
+      resultValue.message = response.body
+    }
```

Comment:
seems like this could be simplified by just altering the message, and for all cases using success = true